src/components/inputs/location/ohri-encounter-location.component.tsx

```diff
@@ -16,14 +16,23 @@ export const OHRIEncounterLocationPicker: React.FC<{ question: OHRIFormField; on
     useContext(OHRIFormContext);
   const [locations, setLocations] = useState([]);
   const [conceptName, setConceptName] = useState('Loading...');
+  const isProcessingSelection = useRef(false);
+  const [inputValue, setInputValue] = useState('');
 
   useEffect(() => {
-    if (question.questionOptions.locationTag) {
-      getLocationsByTag(question.questionOptions.locationTag.trim().split(' ').join('%20')).subscribe(
+    const fetchLocations = () => {
+      const locationTag = question.questionOptions.locationTag;
```

Comment:
This could be a possible failure point if the `locationTag` property does not exist on the field.